### DISCUSSION TITLE
Missing updater notification

### DISCUSSION THREAD

**Discussion by mnaiman** [nextcloud/all-in-one#3465] [Ideas] [answered]
This commit removed plugin for updater notification.
https://github.com/nextcloud/all-in-one/commit/c50132a27ac927d4caa23b777f1454eefed4c4ca#diff-64a67b16c99ece72e02daa4843049bcc16ef3ee1576473390fbf5a5583cff02dR217

Im not using it for update (container) but to have info, there is new version and now missing that feature.

### Describe the solution you'd like
Remove that row, to keep plugin, but disabled (it is already disabled by run.sh)
https://github.com/nextcloud/all-in-one/commit/c50132a27ac927d4caa23b777f1454eefed4c4ca#diff-64a67b16c99ece72e02daa4843049bcc16ef3ee1576473390fbf5a5583cff02dR217

**Comment by szaimen** [comment, marked as answer]
Hi, we disabled the updatenotifications app because it shows wrong update informations because AIO handles its own updates and also sends its own update notifications. So there are no plans to make it available again.

**Comment by mnaiman** [comment]
Disabling is fine, but why to remove it?
It cant be installed again or enabled by admin by occ because it is integral part of nextcloud, so it is not in app store.

**Reply by szaimen** [reply]
Can you explain again why you need this app? What feature of it do you need specifically?

**Comment by mnaiman** [comment]
I want to use built in notification system, about update available and built in system that evaluates incompatible apps (like in screenshot).
Im not performing update itself via this app, but using aio docker.
![image](https://github.com/nextcloud/all-in-one/assets/2264442/95d80c17-4494-445c-abcb-54a38286b8e3)

**Reply by szaimen** [reply]
> built in notification system, about update available and built in system that evaluates incompatible apps

As I said both will report wrong update versions so there are not plans to make this available again...

**Reply by mnaiman** [reply]
in past it was disabled and it was fine. now to re-install it to running instance is painful :/
